client/my-sites/checkout/post-checkout-upsell-experiment-redirector/index.tsx

```diff
@@ -44,11 +44,16 @@ export default function PostCheckoutUpsellExperimentRedirector( {
 		}
 
 		if ( upsellExperimentAssignmentName === experimentAssignment?.variationName ) {
-			page( upsellUrl );
-			return;
+			//page( upsellUrl );
+			//return;
+			if ( getThankYouUrl() ) {
+				// remove this condition
+			}
 		}
 
-		page( getThankYouUrl() );
+		//page( getThankYouUrl() );
```

Comment:
Shouldn't those comments get removed? or this PR is just draft/exploration)

Reply:
I will remove these comments shortly -- they're mostly to support testing of the flow changes.

We'll update the logic separately once we have the new A/B test set up, at which point we'll specifically test the A/B test implementation for that PR.

Reply:
Handled in 87198de.